src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -434,30 +448,19 @@ int CreateRunRequest(int argc, char** argv, const char** env,
   req->add_command("run_remote");
   req->add_command("--name");
   req->add_command(cmd_id);
-  char* invocation_id = getenv("INVOCATION_ID");
-  if (invocation_id != nullptr) {
-    req->add_command("--invocation_id");
-    req->add_command(invocation_id);
-  }
-  char* accept_cached = getenv("ACCEPT_CACHED");
-  if (accept_cached != nullptr) {
-    req->add_command("--accept_cached");
-    req->add_command(accept_cached);
-  }
-  char* save_exec_data = getenv("SAVE_EXECUTION_DATA");
-  if (save_exec_data != nullptr) {
-    req->add_command("--save_execution_data");
-    req->add_command(save_exec_data);
-  }
+  req->add_command("--invocation_id");
```

Comment:
I think you still need to check whether it's empty.

Reply:
Done